### ISSUE TITLE
Plugin drupal/console-extend-plugin could not be initialized

### ISSUE THREAD

**Issue by jaymeh** [hechoendrupal/drupal-console#3817]
## [console] [UnexpectedValueException]

`Plugin drupal/console-extend-plugin could not be initialized, class not found: Drupal\Console\Composer\Plugin\Extender`

![image](https://user-images.githubusercontent.com/18261676/36901863-7ee8591c-1e20-11e8-8274-80fc90ad1928.png)

### Problem/Motivation
Issue was found whilst trying to add console to a fresh install of Drupal 8.4.5. (Running PHP 7.1.13)

### How to reproduce
Install clean version of Drupal 8.4.5 and try to install console using the following command.

```composer require drupal/console:~1.0 --prefer-dist --optimize-autoloader ```

### Drupal Details
- Drupal Version: 8.4.5.
- Console version N/A.
- Console Launcher version: 1.6.1.

**Comment by jmolivas** [comment]
@jaymeh Which OS are you using?

**Comment by jaymeh** [comment]
@jmolivas Sorry, OSX 10.13.3

**Comment by briantully** [comment]
+1 same here with 10.11.6

**Comment by briantully** [comment]
If there are known system/OS version incompatibilities, why not include them in the README?
I lost a full day just trying to get drupal console to work :(

**Comment by jmolivas** [comment]
@jaymeh @briantully DrupalConsole is fully compatible with OSX.

Are you installing in the root project directory?. 

Make sure you are not installing it on `web` or `docroot` if using DrupalComposer or Lightning.

Also, make sure there is no empty `composer.json` file at `web` or `docroot`.

**Comment by briantully** [comment]
I've tried installing Drupal Console several ways according to the docs, all with the same result.

First attempt was the "global" method, i.e., downloading the drupal.phar and moving it to /usr/local/bin as per https://drupalconsole.com/articles/how-to-install-drupal-console and "Console Launcher" method - https://docs.drupalconsole.com/en/getting/launcher.html 

Second attempt was through local composer as per https://docs.drupalconsole.com/en/getting/composer.html

When I try to do the 'local composer' approach, I get the following error:

```
Gathering patches for root package.
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - drupal/console 1.8.0 requires drupal/console-core 1.8.0 -> satisfiable by drupal/console-core[1.8.0].
    - Conclusion: don't install drupal/console-core 1.8.0
    - drupal/console 1.7.0 requires drupal/console-core 1.7.0 -> satisfiable by drupal/console-core[1.7.0].
    - Conclusion: don't install drupal/console-core 1.7.0
    - drupal/console 1.6.1 requires drupal/console-core 1.6.1 -> satisfiable by drupal/console-core[1.6.1].
    - Conclusion: don't install drupal/console-core 1.6.1
    - drupal/console 1.6.0 requires drupal/console-core 1.6.0 -> satisfiable by drupal/console-core[1.6.0].
    - Conclusion: don't install drupal/console-core 1.6.0
    - drupal/console 1.5.0 requires drupal/console-core 1.5.0 -> satisfiable by drupal/console-core[1.5.0].
    - Conclusion: don't install drupal/console-core 1.5.0
    - drupal/console 1.4.0 requires drupal/console-core 1.4.0 -> satisfiable by drupal/console-core[1.4.0].
    - Conclusion: don't install drupal/console-core 1.4.0
    - drupal/console 1.3.2 requires drupal/console-core 1.3.2 -> satisfiable by drupal/console-core[1.3.2].
    - Conclusion: don't install drupal/console-core 1.3.2
    - drupal/console 1.3.1 requires drupal/console-core 1.3.1 -> satisfiable by drupal/console-core[1.3.1].
    - Conclusion: don't install drupal/console-core 1.3.1
    - drupal/console 1.3.0 requires drupal/console-core 1.3.0 -> satisfiable by drupal/console-core[1.3.0].
    - Conclusion: don't install drupal/console-core 1.3.0
    - drupal/console 1.2.0 requires drupal/console-core 1.2.0 -> satisfiable by drupal/console-core[1.2.0].
    - Conclusion: don't install drupal/console-core 1.2.0
    - drupal/console 1.1.1 requires drupal/console-core 1.1.1 -> satisfiable by drupal/console-core[1.1.1].
    - Conclusion: don't install drupal/console-core 1.1.1
    - drupal/console 1.1.0 requires drupal/console-core 1.1.0 -> satisfiable by drupal/console-core[1.1.0].
    - Conclusion: don't install drupal/console-core 1.1.0
    - drupal/console 1.0.2 requires drupal/console-core 1.0.2 -> satisfiable by drupal/console-core[1.0.2].
    - Conclusion: don't install drupal/console-core 1.0.2
    - drupal/console 1.0.1 requires drupal/console-core 1.0.1 -> satisfiable by drupal/console-core[1.0.1].
    - Conclusion: don't install drupal/console-core 1.0.1
    - drupal/console 1.0.0 requires drupal/console-core 1.0.0 -> satisfiable by drupal/console-core[1.0.0].
    - drupal/console 1.0.0-rc20 requires drupal/console-core 1.0.0-rc20 -> satisfiable by drupal/console-core[1.0.0-rc20].
    - drupal/console 1.0.0-rc21 requires drupal/console-core 1.0.0-rc21 -> satisfiable by drupal/console-core[1.0.0-rc21].
    - drupal/console 1.0.0-rc22 requires drupal/console-core 1.0.0-rc22 -> satisfiable by drupal/console-core[1.0.0-rc22].
    - drupal/console 1.0.0-rc23 requires drupal/console-core 1.0.0-rc23 -> satisfiable by drupal/console-core[1.0.0-rc23].
    - drupal/console 1.0.0-rc24 requires drupal/console-core 1.0.0-rc24 -> satisfiable by drupal/console-core[1.0.0-rc24].
    - drupal/console 1.0.0-rc25 requires drupal/console-core 1.0.0-rc25 -> satisfiable by drupal/console-core[1.0.0-rc25].
    - drupal/console 1.0.0-rc26 requires drupal/console-core 1.0.0-rc26 -> satisfiable by drupal/console-core[1.0.0-rc26].
    - drupal/console-core 1.0.0 requires webflo/drupal-finder ^0.3.0 -> satisfiable by webflo/drupal-finder[0.3.0].
    - drupal/console-core 1.0.0-rc20 requires webflo/drupal-finder ^0.3.0 -> satisfiable by webflo/drupal-finder[0.3.0].
    - drupal/console-core 1.0.0-rc21 requires webflo/drupal-finder ^0.3.0 -> satisfiable by webflo/drupal-finder[0.3.0].
    - drupal/console-core 1.0.0-rc22 requires webflo/drupal-finder ^0.3.0 -> satisfiable by webflo/drupal-finder[0.3.0].
    - drupal/console-core 1.0.0-rc23 requires webflo/drupal-finder ^0.3.0 -> satisfiable by webflo/drupal-finder[0.3.0].
    - drupal/console-core 1.0.0-rc24 requires webflo/drupal-finder ^0.3.0 -> satisfiable by webflo/drupal-finder[0.3.0].
    - drupal/console-core 1.0.0-rc25 requires webflo/drupal-finder ^0.3.0 -> satisfiable by webflo/drupal-finder[0.3.0].
    - drupal/console-core 1.0.0-rc26 requires webflo/drupal-finder ^0.3.0 -> satisfiable by webflo/drupal-finder[0.3.0].
    - Conclusion: don't install webflo/drupal-finder 0.3.0
    - Conclusion: don't install drupal/console 1.0.0-rc19
    - Conclusion: don't install drupal/console 1.0.0-rc18
    - Conclusion: don't install drupal/console 1.0.0-rc17
    - drupal/console 1.0.0-rc15 requires doctrine/collections 1.3.0 -> satisfiable by doctrine/collections[v1.3.0].
    - drupal/console 1.0.0-rc16 requires doctrine/collections 1.3.0 -> satisfiable by doctrine/collections[v1.3.0].
    - Conclusion: don't install doctrine/collections v1.3.0
    - Conclusion: don't install drupal/console 1.0.0-rc14
    - Conclusion: don't install drupal/console 1.0.0-rc13
    - Conclusion: don't install drupal/console 1.0.0-rc12
    - Conclusion: don't install drupal/console 1.0.0-rc11
    - Conclusion: don't install drupal/console 1.0.0-rc10
    - Conclusion: don't install drupal/console 1.0.0-rc9
    - Conclusion: don't install drupal/console 1.0.0-rc8
    - Conclusion: don't install drupal/console 1.0.0-rc7
    - Conclusion: don't install drupal/console 1.0.0-rc6
    - Conclusion: don't install drupal/console 1.0.0-rc5
    - Conclusion: don't install drupal/console 1.0.0-rc4
    - Conclusion: don't install drupal/console 1.0.0-rc3
    - Conclusion: don't install drupal/console 1.0.0-rc2
    - Conclusion: don't install drupal/console 1.0.0-rc1
    - Conclusion: don't install drupal/console 1.0.0-beta5
    - Conclusion: don't install drupal/console 1.0.0-beta4
    - Conclusion: don't install drupal/console 1.0.0-beta3
    - Conclusion: don't install drupal/console 1.0.0-beta2
    - Conclusion: don't install drupal/console 1.0.0-beta1
    - Conclusion: don't install drupal/console 1.0.0-alpha2
    - Conclusion: remove symfony/config v3.4.6
    - Installation request for symfony/debug (locked at v4.0.6) -> satisfiable by symfony/debug[v4.0.6].
    - Installation request for webflo/drupal-finder (locked at 1.1.0) -> satisfiable by webflo/drupal-finder[1.1.0].
    - Installation request for doctrine/collections (locked at v1.5.0) -> satisfiable by doctrine/collections[v1.5.0].
    - Installation request for doctrine/annotations (locked at v1.6.0) -> satisfiable by doctrine/annotations[v1.6.0].
    - Installation request for drupal/console ~1.0 -> satisfiable by drupal/console[1.0.0, 1.0.0-alpha1, 1.0.0-alpha2, 1.0.0-beta1, 1.0.0-beta2, 1.0.0-beta3, 1.0.0-beta4, 1.0.0-beta5, 1.0.0-rc1, 1.0.0-rc10, 1.0.0-rc11, 1.0.0-rc12, 1.0.0-rc13, 1.0.0-rc14, 1.0.0-rc15, 1.0.0-rc16, 1.0.0-rc17, 1.0.0-rc18, 1.0.0-rc19, 1.0.0-rc2, 1.0.0-rc20, 1.0.0-rc21, 1.0.0-rc22, 1.0.0-rc23, 1.0.0-rc24, 1.0.0-rc25, 1.0.0-rc26, 1.0.0-rc3, 1.0.0-rc4, 1.0.0-rc5, 1.0.0-rc6, 1.0.0-rc7, 1.0.0-rc8, 1.0.0-rc9, 1.0.1, 1.0.2, 1.1.0, 1.1.1, 1.2.0, 1.3.0, 1.3.1, 1.3.2, 1.4.0, 1.5.0, 1.6.0, 1.6.1, 1.7.0, 1.8.0].
    - Conclusion: don't install symfony/config v3.4.6
    - drupal/console 1.0.0-alpha1 requires symfony/finder ~2.8 -> satisfiable by symfony/finder[2.8.x-dev, v2.8.0, v2.8.0-BETA1, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.19, v2.8.2, v2.8.20, v2.8.21, v2.8.22, v2.8.23, v2.8.24, v2.8.25, v2.8.26, v2.8.27, v2.8.28, v2.8.29, v2.8.3, v2.8.30, v2.8.31, v2.8.32, v2.8.33, v2.8.34, v2.8.35, v2.8.36, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - symfony/finder 2.8.x-dev conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.0 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.0-BETA1 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.1 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.10 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.11 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.12 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.13 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.14 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.15 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.16 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.17 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.18 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.19 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.2 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.20 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.21 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.22 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.23 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.24 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.25 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.26 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.27 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.28 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.29 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.3 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.30 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.31 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.32 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.33 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.34 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.35 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.36 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.4 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.5 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.6 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.7 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.8 conflicts with symfony/config[v3.4.6].
    - symfony/finder v2.8.9 conflicts with symfony/config[v3.4.6].
    - Installation request for symfony/config (locked at v3.4.6) -> satisfiable by symfony/config[v3.4.6].
```

**Comment by gilbertomangones** [comment]
Hi, I tried on a Drupal installed by composer.

```drupalspanish mac$ composer require drupal/console:~1.0 --prefer-dist --optimize-autoloader```

![x1](https://user-images.githubusercontent.com/6983125/65190189-b8ccd980-da37-11e9-9c14-b591a44df23e.png)

Drupal Console version 1.9.3

**Comment by enzolutions** [comment]
Looks like the errors was fixed and is not available in version 1.9.3
